src/Components/LoggedInPage.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
```

Comment:
![Screen Shot 2023-06-02 at 12 07 58 PM](https://github.com/LetsMesh/Site/assets/75825603/9c2665e0-9732-45f6-bf53-cad350540df8)
If you want to change the font of the text to match the figma, the the cocogoose font should be in the main branch